src/main/flight/pid.c

```diff
@@ -93,6 +93,21 @@ void pidResetErrorGyro(void)
 
 const angle_index_t rcAliasToAngleIndexMap[] = { AI_ROLL, AI_PITCH };
 
+
+// Low Pass filter for PID controllers
+void pt1_element(int axis, float *delta, uint8_t f_cut, int type) {
+
+	float dT = (float)cycleTime * 0.000001f;
+
+    float    RC;
+    static float    lastDelta[][3] = {{ 0.0f, 0.0f, 0.0f }};
+
```

Comment:
@borisbstyle : Most likely you will overwrite some other function-static variable. For my compiler, ledsInRingCount, ledGridHeight, spi1ErrorCount, i2cErrorCount are placed next in memory, but that is probably dependent on version.
This is extremely dangerous - this bug may be hidden for a long time and it is quite hard to debug this.
And the result may be catastrophic ...